x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -18,29 +18,35 @@ import { FormattedMessage, FormattedNumber } from '@kbn/i18n-react';
 import { INPUT_THROTTLE_DELAY_MS } from '../../../shared/constants/timers';
 import { DataPanel } from '../../../shared/data_panel/data_panel';
 
-import { EnterpriseSearchContentPageTemplate } from '../layout/page_template';
+import { EnterpriseSearchEnginesPageTemplate } from '../layout/engines_page_template';
 
 import { EnginesListTable } from './components/tables/engines_table';
+import { CreateEngineFlyout } from './create_engine_flyout';
 import { DeleteEngineModal } from './delete_engine_modal';
 import { EnginesListLogic } from './engines_list_logic';
 
 export const EnginesList: React.FC = () => {
-  const { fetchEngines, onPaginate, openDeleteEngineModal } = useActions(EnginesListLogic);
-  const { meta, results } = useValues(EnginesListLogic);
-  const [searchQuery, setSearchValue] = useState('');
+  const {
+    closeEngineCreate,
+    fetchEngines,
+    onPaginate,
+    openEngineCreate,
+    openDeleteEngineModal,
+    setSearchQuery,
+  } = useActions(EnginesListLogic);
+  const { meta, results, createEngineFlyoutOpen, searchQuery } = useValues(EnginesListLogic);
+
   const throttledSearchQuery = useThrottle(searchQuery, INPUT_THROTTLE_DELAY_MS);
```

Comment:
suggestion(non-blocking): use a kea breakpoint instead of a component throttle, so we automatically throttle everywhere we make this call instead of just on this component. As above, we could add a delay option to the createEngineApi logic for this as well.